tests/TestCase/Http/BaseApplicationTest.php

```diff
@@ -196,9 +196,7 @@ public function testPluginBootstrapRecursivePlugins()
             [$this->path]
         );
         $app->addPlugin('ParentPlugin');
-        $this->deprecated(function () use ($app) {
-            $app->pluginBootstrap();
-        });
+        $app->pluginBootstrap();
```

Comment:
This call was wrapped in `deprecated()` earlier shouldn't the method be removed all together in 4.x?

Reply:
no
deprecated class (Plugin::load ) replaced in bellow  commit 
 https://github.com/cakephp/cakephp/commit/d78f0e23620a7259534e7b95fea2ccd4ad8f6c36#diff-7dfbdbac6f63af358352d41a16afddd9
so we can remove wrapper here